awswrangler/distributed/ray/modin/_utils.py

```diff
@@ -26,6 +27,14 @@ def _block_to_df(
     return _table_to_df(table=block._table, kwargs=to_pandas_kwargs)  # pylint: disable=protected-access
 
 
+def _from_df(df: Union[pd.DataFrame, modin_pd.DataFrame]) -> Dataset[Any]:
```

Comment:
Small nitpick: Could we call it something like `_ray_dataset_from_df`, so the name makes it clearer what we're converting to.

Reply:
Sure, why not. Done